### PR TITLE
[SH9392] New Feaure adding methods mutations, inside redux for send emails from backend

### PR DESCRIPTION
**Pull request by 92Sam**
https://app.shortcut.com/masterventures/story/9332/as-a-front-end-developer-i-to-call-the-mutations-for-sending-emails-after-agreement-is-anchored-or-signed

### CODE REVIEW

**Review by CoolHandLuke88 — APPROVED** [review]

File: pages/new-agreement.tsx [hunk 1]
```diff
@@ -231,6 +232,23 @@ const NewAgreement: NextPage<NewAgreementProps> = ({ templateTypeCode }) => {
...
+        documentId: tx.hash,
+        network: NetworkName.RINKEBY
+      }))
+
+      dispatch(sendEmailSigned({
+        agreementTitle: templateTitle,
+        documentId: tx.hash,
+        network: NetworkName.RINKEBY,
+        proponentDid: profile.did
+      }))
```
Comment (zubietaroberto) [inline pages/new-agreement.tsx:250]: Remove this from here. This action has to happen in the "Sign" action of the Agreement Details.

**Review by zubietaroberto — CHANGES_REQUESTED** [review]

File: pages/new-agreement.tsx (hunk 1 above)
Comment (92Sam) [reply pages/new-agreement.tsx:250]: WIP

**Review by CoolHandLuke88 — APPROVED** [review]
Approved, as long as feedback is addressed.

File: pages/agreements.tsx [hunk 2]
```diff
@@ -121,10 +121,18 @@ const Agreements: React.FC = () => {
...
     agreementToUpdate.event.status = agreementStatus.ACCEPTED;
     helper.newFormatDate(new Date());
     dispatch(updateAgreement(currentAgreement?.event.cid, agreementToUpdate));
+
+    dispatch(sendEmailSigned({
+      agreementTitle: agreementToUpdate.data.documentName,
+      documentId: agreementToUpdate.transactionHash,
+      network: networkEnum,
+      proponentDid: agreementToUpdate.data.fromSigner,
+    }))
```
Comment (zubietaroberto) [inline pages/agreements.tsx:135]: This goes in the AgreementDetails page, not here.

**Review by zubietaroberto — CHANGES_REQUESTED** [review]

**Review by CoolHandLuke88 — DISMISSED** [review]

**Review by zubietaroberto — APPROVED** [review]
Approved

**Review by CoolHandLuke88 — APPROVED** [review]

**Pull request merged**